modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataLayoutResourceImpl.java

```diff
@@ -475,6 +486,48 @@ private long _getDDMStructureVersionId(long deDataDefinitionId)
 			"$[\"pages\"][*][\"rows\"][*][\"columns\"][*][\"fieldNames\"][*]");
 	}
 
+	private void _initThemeDisplay(
```

Comment:
Can we do something like [this](https://github.com/liferay/liferay-portal/blob/f1dbd869c1036f3bc6379585b4c2a0293877d389/modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/util/RenderedContentValueUtil.java#L99) instead?

Reply:
Sure thing, @jeyvison